simtbx/diffBragg/hopper_ensemble_utils.py

```diff
@@ -641,16 +647,17 @@ def load_inputs(pandas_table, params, exper_key="exp_name", refls_key='predictio
         shot_modeler.exper_name = exper_name
         shot_modeler.refl_name = refl_name
         shot_modeler.rank = COMM.rank
-        if params.refiner.load_data_from_refl:
-            gathered = shot_modeler.GatherFromReflectionTable(expt, refls, sg_symbol=params.space_group)
-            MAIN_LOGGER.debug("tried loading from reflection table")
-        else:
-            gathered = shot_modeler.GatherFromExperiment(expt, refls, sg_symbol=params.space_group)
-            MAIN_LOGGER.debug("tried loading data from expt table")
-        if not gathered:
-            raise IOError("Failed to gather data from experiment %s", exper_name)
-        else:
+        try:
+            if params.refiner.load_data_from_refl:
+                gathered = shot_modeler.GatherFromReflectionTable(expt, refls, sg_symbol=params.space_group)
+                MAIN_LOGGER.debug("tried loading from reflection table")
+            else:
+                gathered = shot_modeler.GatherFromExperiment(expt, refls, sg_symbol=params.space_group)
+                MAIN_LOGGER.debug("tried loading data from expt table")
             MAIN_LOGGER.debug("successfully loaded data")
+        except RuntimeError:
+            MAIN_LOGGER.critical("Failed to gather data from experiment %s; skipping.", exper_name)
+            continue
```

Comment:
Got it, one way I got around this in the past was by pre-filtering the input pandas pickle, removing rows if they pointed to reflection tables with too few or too many reflections.. But this is ok to keep here